caffe-grid/src/main/python/examples/README-coco.md

```diff
@@ -0,0 +1,121 @@
+Steps to run the COCO dataset for Image Captioning
+==================================================
+##### (1) Env setup
+    export CAFFE_ON_SPARK=/Users/mridul/bigml/CaffeOnSpark
+    export DYLD_LIBRARY_PATH=${CAFFE_ON_SPARK}/caffe-public/distribute/lib:${CAFFE_ON_SPARK}/caffe-distri/distribute/lib
+    export DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}:/usr/local/cuda/lib:/usr/local/mkl/lib/intel64/
+    export LD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}
+    export SPARK_HOME=/Users/mridul/bigml/spark-1.6.0-bin-hadoop2.6
```

Comment:
Let's refer to our GetStarted guides (https://github.com/yahoo/CaffeOnSpark/wiki/GetStarted_standalone_osx). We should inform our users to set up CAFFE_ON_SPARK and SPARK_HOME etc from our guides. Don't want to expose your own environments to our users.